aea/multiplexer.py

```diff
@@ -342,9 +342,9 @@ async def disconnect(self) -> None:
                 self.connection_status.set(ConnectionStates.disconnecting)
                 await asyncio.wait_for(self._stop(), timeout=60)
                 self.logger.debug("Multiplexer disconnected.")
-            except (CancelledError) as e:
-                self.logger.exception("Exception on disconnect:")
-                raise AEAConnectionError("Failed to disconnect the multiplexer.") from e
+            except CancelledError:  # pragma: nocover
+                self.logger.debug("Multiplexer.disconnect cancellation!")
+                raise
```

Comment:
What problem does this relate to? Looks good, just wondering

Reply:
not a problem, just log that disconnect cancelled and exception propagated,
it's not assumed that disconnect should be cancelled, but it's possible